src/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -131,8 +137,11 @@ private MemberDependency GetMemberReferenceMemberDependency(MemberReference memb
                 definedInAssemblyIdentity = _reader.FormatAssemblyInfo(memberRefInfo.ParentType.DefinedInAssembly.Value);
             }
             // If no assembly is set, then the type is either a primitive type or it's in the current assembly.
```

Comment:
Not sure I understand the comment here... Is it possible for a parent to be a primitive type and defined in the current assembly (that could be a user assembly).